system/cachespeed/cachespeed_main.c

```diff
@@ -201,7 +201,7 @@ static void dcache_speed_test(FAR void *begin, size_t memset_size,
   uint32_t flush_cost_time;
   uint32_t cnt;
   uint32_t pt;
-  irqstate_t flags;
+  irqstate_t flags = 0;
```

Comment:
I do not think this init is needed here. What exact warning is reported here? flags variable is unconditionally initialized at line 217

Reply:
Sorry, there was a mistake in this change